Final2-Another Candy Crush Clone/edu/kit/informatik/matchthree/tests/MoveFlipDownTest.java

```diff
@@ -73,12 +73,14 @@ public void applyAndReverse() {
     public void getAffectedPositions() {
 
         Board board = new MatchThreeBoard(Token.set("AB"), 3, 3);
-        Move flipDown = new MoveFactoryImplementation().flipDown(Position.at(0, 2));
-        
+        // changed initial position, as the move would have been illegal
+        // otherwise (or was this intended?)
```

Comment:
you're right - wasn't intended; unclear if exceptions should be thrown if getAffectedPositions() would return positions which aren't on the board